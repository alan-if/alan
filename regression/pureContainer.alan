The air Isa entity
  Container Taking thing.
End The air.

The spring Isa thing In air
End The spring.

The l Isa location
End The l.

Start At l.
  If spring Here Then "Spring is in the air." End If.

