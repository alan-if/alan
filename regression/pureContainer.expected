########## pureContainer ##########

        No warnings or errors detected.


Spring is in the air.


L

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
